src/io/opendss.jl

```diff
@@ -1231,6 +1244,7 @@ function _create_vbranch!(pmd_data, f_bus::Int, t_bus::Int; name="", source_id="
     end
     vbranch["angmin"] = -_PMs.MultiConductorVector(ones(ncnd))*60
     vbranch["angmax"] = _PMs.MultiConductorVector(ones(ncnd))*60
+    vbranch["rate_a"] = rate_a
```

Comment:
Instead of creating a new `rate_a` kwarg with a default that's complex like that, I would use `get(kwargs, :rate_a, rate_a_default)` on L1247.

Reply:
I like that pattern, I should use it more.